spec/unleash/client_spec.rb

```diff
@@ -155,6 +157,13 @@
       unleash_client.is_enabled?('toggleName', {}, true)
     ).to eq(true)
 
+    expect(
+      unleash_client.get_variant('toggleName', Unleash::Context.new(user_id: 42))
+    ).to eq(Unleash::Variant.new(name: 'a', enabled: true, payload: {
+      'type' => 'string',
+      'value' => ''
+    }))
```

Comment:
There appeared to be no tests covering any aspect of the variant logic, so this is the first, and by no means is it comprehensive, but it should generally cover "happy path" at a minimum.

Reply:
Also worth to note that you only match the Variant "`a`" because of the hashing of the `user_id: 42`. different values in the context could give different results for which variant gets picked.